client/src/themes/theme.ts

```diff
@@ -10,6 +10,12 @@ export const theme = createMuiTheme({
     body1: {
       color: '#f14140',
     },
+    h2: {
```

Comment:
Great work using MUI's theme the way it's supposed to be used 👍🏽

Reply:
Thank you.